ansible/roles/mistral/vars/main.yml

```diff
@@ -0,0 +1,9 @@
+mistral_branches:
```

Comment:
I wish we could provide this information in some common place so it doesn't need to be hard-coded.

Or ideally, we would just assume, that going forward, there should be a branch for each version which follows a standard naming format so we don't need ifs anymore (we don't support versions older than 0.8 anymore anyway).